ludwig/schema/features/sequence_feature.py

```diff
@@ -66,6 +67,13 @@ class SequenceOutputFeatureConfig(BaseOutputFeatureConfig, SequenceOutputFeature
     """SequenceOutputFeatureConfig is a dataclass that configures the parameters used for a sequence output
     feature."""
 
+    default_validation_metric: str = schema_utils.StringOptions(
```

Comment:
Is this metric parameter on every output feature?

Reply:
Yes, it was previously stored as a class attribute on the output_features themselves but moved it to the schema in the spirit of SSoT.

Reply:
If it's required then I'd say stick it on some inherited base class.

Reply:
Definitely will do! Though I would still have to leave it on each of the individual output features because each output feature has a different default validation metric so I would just set the one on the base class to none.